From e5855850ae6dc17e0fd92b2b2b4ed057b1eb3bf6 Mon Sep 17 00:00:00 2001
From: Wright Feng <wright.feng@cypress.com>
Date: Tue, 29 Sep 2020 22:34:31 -0500
Subject: [PATCH 4/4] wifi: Disable debug ring buffer flags because not support
 currently

FMAC does not support DEBUG_RING_BUFFER_VENDOR_DATA currently,
so we remove it from debugging flags in wifihal.

Signed-off-by: Wright Feng <wright.feng@cypress.com>

JIRA: SWLINUX-1792
---
 wifi/1.3/default/hidl_struct_util.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/wifi/1.3/default/hidl_struct_util.cpp b/wifi/1.3/default/hidl_struct_util.cpp
index d305c0997..68f9a5fe4 100644
--- a/wifi/1.3/default/hidl_struct_util.cpp
+++ b/wifi/1.3/default/hidl_struct_util.cpp
@@ -157,7 +157,8 @@ bool convertLegacyFeaturesToHidlChipCapabilities(
 
     // There are no flags for these 3 in the legacy feature set. Adding them to
     // the set because all the current devices support it.
-    *hidl_caps |= HidlChipCaps::DEBUG_RING_BUFFER_VENDOR_DATA;
+    // Cypress FMAC does not support debug ring buffer, will have it soon.
+    // *hidl_caps |= HidlChipCaps::DEBUG_RING_BUFFER_VENDOR_DATA;
     *hidl_caps |= HidlChipCaps::DEBUG_HOST_WAKE_REASON_STATS;
     *hidl_caps |= HidlChipCaps::DEBUG_ERROR_ALERTS;
     return true;
-- 
2.25.0

